package.json

```diff
@@ -78,6 +78,7 @@
 		"migrate-mongoose": "^4.0.0",
 		"mixwith": "^0.1.1",
 		"moment": "^2.19.2",
+		"moment-timezone": "^0.5.31",
```

Comment:
Don't need it here, because timezone conversion is handled by the client

Reply:
done ;)